public/app/features/alerting/unified/__snapshots__/PanelAlertTabContent.test.tsx.snap

```diff
@@ -0,0 +1,117 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`PanelAlertTabContent Will render alerts belonging to panel and a button to create alert from panel queries 1`] = `
```

Comment:
💬 I've moved the output of one of the tests in `PanelAlertTabContent` that asserts the data structure of a default alert to a snapshot here as it's easier to work with and update when we change it later (to a reduce + threshold expression).